test/legacy/sample.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<head>
+    <meta name='ocr-id' value='OCRopus Revision: 312'>
+    <meta name='ocr-recognized' value='lines text'>
+    <meta name='DC.creator' value='James T. Kirk'>
+    <meta name='DC.title' value='Nomad'>
+    <meta name='DC.publisher' value='NASA'>
+</head>
```

Comment:
Why do we duplicate this file here again? Can we not use the same copy of the hocr file as in the test `test/hocr-check`?